build-utils/webpack.common.js

```diff
@@ -0,0 +1,66 @@
+const commonPaths = require('./common-paths')
+const webpack = require('webpack')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+
+const config = {
+  entry: {
+    app: `${commonPaths.appEntry}/index.tsx`,
+    vendor: ['semantic-ui-react']
+  },
+  output: {
+    path: commonPaths.outputPath,
+    publicPath: '/'
+  },
+  resolve: {
+    extensions: [".js", ".jsx", ".json", ".ts", ".tsx"],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(ts|tsx)$/,
+        loader: "awesome-typescript-loader",
+      },
+      // {
+      //   enforce: "pre",
+      //   test: /\.js$/,
+      //   loader: "source-map-loader",
+      // },
+      {
+        // Load fonts
+        test: /\.woff($|\?)|\.woff2($|\?)|\.ttf($|\?)|\.eot($|\?)|\.svg($|\?)/,
+        use: 'url-loader',
+      },
+      {
+        // Load other files, images etc
+        test: /\.(png|j?g|gif|ico)?$/,
+        use: 'url-loader',
+      }
+    ]
+  },
+  optimization: {
```

Comment:
Was wondering what this would do ?

Reply:
This is called [Code splitting](https://webpack.js.org/guides/code-splitting/) and it's used to separate the `semantic-ui-react` vendor in another chunk. Not sure exactly what the one for styles is doing. It would be something similar for css files using the `semantic-ui-less` in a separated chunk. I got this instructions from this [tutorial](https://medium.com/free-code-camp/learn-webpack-for-react-a36d4cac5060#4e46). 
The results when I run `yarn build` with using the code splitting:
<img width="1085" alt="Screen Shot 2020-09-17 at 10 23 05 AM" src="https://user-images.githubusercontent.com/16461988/93398954-a4345300-f8d0-11ea-9786-cbc0f2b6466c.png">
And without using the code splitting:
<img width="891" alt="Screen Shot 2020-09-17 at 10 30 15 AM" src="https://user-images.githubusercontent.com/16461988/93399024-d04fd400-f8d0-11ea-8511-94a748f66bcf.png">